src/pymor/discretizations/iosys.py

```diff
@@ -157,6 +157,11 @@ def mag_plot(cls, sys_list, plot_style_list=None, w=None, ord=None, dB=False, Hz
         return fig, ax
 
 
+_DEFAULT_ME_SOLVER = 'slycot' if config.HAVE_SLYCOT else \
+                     'pymess' if config.HAVE_PYMESS else \
```

Comment:
`pymess` should be prefered over `slycot`, since `slycot` can't handle large equations.